### PR TITLE
Story/tg 12 heroku

### PR DESCRIPTION
**Pull request by disist**
Integration with heroku

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/12" target="_blank">ReviewNinja

File: .travis.yml [hunk 1]
```diff
@@ -16,4 +15,12 @@ script:
...
\ No newline at end of file
+  - npm run report-coverage
+deploy:
+  provider: heroku
+  api_key:
+    secure: LZUT2N961Yb7Wjp3jkrPOtBT803z8O6KVenrdFeFwt7+GD4vVgywR0wrBYw6n1yLzzkuJOdvjZe2dmEj6DNH8TkxOi/DemaSxDz8XzbYWdoB8ietVikzAoDfrcXuyrnz9qn5gzLH5RnPJ/Z5zzgnONimgDuUlqb2avTPyrUMZ2DE6opBEMQqfIAjKsAlF1UXpWX255BIdwnf2FEtHp8cYd2BNga9nuXJ6AfCVapfiD9H06cDxOR6tub4jOTdd9k6mJ72ShdHz/3tSLESSnFqo+m0wxnVw7SpMEj4CBw5V2mS/k3K9DIiPO2bQcTxUHYnG/OH7J97aS6eZecbM7rqxgyggQj9TcvwnO6PGnRxmy1XE5pOVcjVzUO1weqJtLXlomzi31JQaiUsKu3XqqMMZ5705ja0r9mB2Qr+OGgiCZbbjERdtkwQCHBAgiGe2WJN/bvl25r9WvO9lPC43W1sGIurlnhGiawU+m0WcfE76zsYjyXbHyHvjbVLeIH9V94FYU/fNEY3f0J/T403NIthQKyFekHCrGu7sFnAFwbeQTdl9eKPD6dsJiU09AQEuCzIRK4QiSEExnft5TDFWHg52ojT0+vQOd6sbxiBdMLERvR7ZQirY0FBSXxRBNWaUVSJuzxiz6/g15CmMWvcvKQ4zwZfmV0wc1HtruiYUrTYJro=
+  app: garage-it-smart-house-api
+  on:
+    repo: garage-it/SmartHouse-backend
+    branch: story/TG-12-heroku
```
Comment (yurii-sorokin) [inline .travis.yml]: master maybe?

File: src/config/env/heroku.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export default {
+    env: 'heroku',
+    db: 'mongodb://garage-it:smarthause@ds017553.mlab.com:17553/garage-it',
+    port: process.env.PORT,
+    seedDB: true,
```
Comment (disist) [inline src/config/env/heroku.js]: my mistake
its will be false

File: .travis.yml (hunk 1 above)
Comment (disist) [inline .travis.yml]: You absolutely right. Thanks!

File: .travis.yml [hunk 3]
```diff
@@ -16,4 +15,11 @@ script:
   - npm test
   - npm run build
 after_script:
-  - npm run report-coverage
\ No newline at end of file
+  - npm run report-coverage
+deploy:
+  provider: heroku
+  api_key:
+    secure: LZUT2N961Yb7Wjp3jkrPOtBT803z8O6KVenrdFeFwt7+GD4vVgywR0wrBYw6n1yLzzkuJOdvjZe2dmEj6DNH8TkxOi/DemaSxDz8XzbYWdoB8ietVikzAoDfrcXuyrnz9qn5gzLH5RnPJ/Z5zzgnONimgDuUlqb2avTPyrUMZ2DE6opBEMQqfIAjKsAlF1UXpWX255BIdwnf2FEtHp8cYd2BNga9nuXJ6AfCVapfiD9H06cDxOR6tub4jOTdd9k6mJ72ShdHz/3tSLESSnFqo+m0wxnVw7SpMEj4CBw5V2mS/k3K9DIiPO2bQcTxUHYnG/OH7J97aS6eZecbM7rqxgyggQj9TcvwnO6PGnRxmy1XE5pOVcjVzUO1weqJtLXlomzi31JQaiUsKu3XqqMMZ5705ja0r9mB2Qr+OGgiCZbbjERdtkwQCHBAgiGe2WJN/bvl25r9WvO9lPC43W1sGIurlnhGiawU+m0WcfE76zsYjyXbHyHvjbVLeIH9V94FYU/fNEY3f0J/T403NIthQKyFekHCrGu7sFnAFwbeQTdl9eKPD6dsJiU09AQEuCzIRK4QiSEExnft5TDFWHg52ojT0+vQOd6sbxiBdMLERvR7ZQirY0FBSXxRBNWaUVSJuzxiz6/g15CmMWvcvKQ4zwZfmV0wc1HtruiYUrTYJro=
```
Comment (kosich) [inline .travis.yml]: is this a secret key? can we store it in public repo?

File: src/config/env/heroku.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export default {
+    env: 'heroku',
+    db: 'mongodb://garage-it:smarthause@ds017553.mlab.com:17553/garage-it',
+    port: process.env.PORT,
+    seedDB: false,
+    mqtt: {
+        port: 12787,
+        hostname: 'm21.cloudmqtt.com',
```
Comment (kosich) [inline src/config/env/heroku.js]: its secret information, cant be stored in public repo

File: .travis.yml (hunk 3 above)
Comment (disist) [inline .travis.yml]: Yep, why not, it's common solution when deploy to heroku from travis. 
Which described in docs 
https://docs.travis-ci.com/user/deployment/heroku/

File: src/config/env/heroku.js (hunk 4 above)
Comment (disist) [inline src/config/env/heroku.js]: !fix
I guess, In this case I will hide this in ```process.env```
What your opinion or suggestion about this?

Comment (kosich) [inline src/config/env/heroku.js]: I don't have experience in that. needs investigation

Comment (disist) [inline src/config/env/heroku.js]: !fixed
In this case (when we deploy with heroku config), I hide this in ```process.env```

**Comment by kosich** [comment]
⭐

**Pull request merged**